.github/workflows/aspect_workflows_deps.bzl

```diff
@@ -0,0 +1,16 @@
+"""Aspect Workflows bazel dependencies"""
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+ASPECT_WORKFLOWS_VERSION = "5.7.0-rc4"
```

Comment:
how will users discover that there's a newer release?
We can't add a test asserting that this version matches their terraform, because we don't plan to release workflows-action at every release, and also they often have their terraform in a separate repo

Reply:
I don't have a good answer for that. Typically customers will not have their Aspect Workflows GitHub Actions configuration in the same repository as their terraform. Updating the vendored workflows if there are fixes and/or new features will have to be part of the upgrade process. We'll need to be careful to keep new versions of Workflows backward compatible with older versions of the re-usable workflow.